catalyst_voices/packages/catalyst_voices_shared/lib/src/common/app_config.dart

```diff
@@ -0,0 +1,31 @@
+/// `AppConfig` holds configuration values for the application.
+/// To pass configuration values to the application,
+/// set the environment variables
+/// with dart define when running the application.
+/// ```
+/// flutter run --dart-define SENTRY_DSN=https://dev.api.example.com --dart-define SENTRY_ENVIRONMENT=qa
+/// ```
+/// To access the environment variables, use the `AppConfig` class.
+/// For example, to get the 'SENTRY_DSN' environment variable,
+///  use `AppConfig.sentryDsn`.
+final class AppConfig {
+  /// Gets the 'SENTRY_DSN' from the environment variables.
+  static String sentryDsn = const String.fromEnvironment(
```

Comment:
Is this on purpose that we don't set sentryDsn as const or final? If defined in this way you can override it from any place in code with:

```
AppConfig.sentryDsn = 'whatever';
```